apps/backend/src/routes/auth/token.ts

```diff
@@ -20,7 +20,15 @@ router.post<null, ApiResponseBody<LoginResponseBody>>(
       return;
     }
 
-    const athleteId = user.data.stravaAuthentication?.athleteId ?? null;
+    const stravaData = user.data.stravaAuthentication
+      ? {
+          athleteId: user.data.stravaAuthentication.athleteId,
+          scopes: {
+            read: user.data.stravaAuthentication.readScope,
+            activityWrite: user.data.stravaAuthentication.activityWriteScope,
+          },
+        }
+      : null;
```

Comment:
Could probably extract this to a shared function, since the exact same thing is done everywhere we return a token – but that can be done separately 🤠